src/hrflow_connectors/connectors/boards/smartrecruiters/actions.py

```diff
@@ -0,0 +1,178 @@
+        # job sections: descriptions and qualifications
+        companyDescription = (
+            data.get("jobAd", {})
+            .get("sections", {})
+            .get("companyDescription", {})
+            .get("text")
+        )
+        jobDescription = (
+            data.get("jobAd", {})
+            .get("sections", {})
+            .get("jobDescription", {})
+            .get("text")
+        )
+        qualification = (
+            data.get("jobAd", {})
+            .get("sections", {})
+            .get("qualifications", {})
+            .get("text")
+        )
+        additional_information = (
+            data.get("jobAd", {})
+            .get("sections", {})
+            .get("additionalInformation", {})
+            .get("text")
+        )
+        job["sections"] = [
+            dict(
+                name="company_description",
+                title="companyDescription",
+                description=companyDescription,
+            ),
+            dict(
+                name="job_description",
+                title="jobDescription",
+                description=jobDescription,
+            ),
+            dict(
+                name="qualifications", title="qualifications", description=qualification
+            ),
+            dict(
+                name="additional_information",
+                title="additionalInformation",
+                description=additional_information,
+            ),
+        ]
+        # job tags
+        status = data.get("status")
+        posting_status = data.get("postingStatus")
+        job_uuid = data.get("id")
+        experience_level = data.get("experienceLevel", {}).get("id")
+        employmentType = data.get("typeOfEmployment", {}).get("id")
+        industry = data.get("industry", {}).get("id")
+        creator = data.get(
+            "creator",
+        )
+        function = data.get("function", {}).get("id")
+        department = data.get("department", {}).get("id")
+        manual = data.get("location", {}).get("manual")
+        remote = data.get("location", {}).get("remote")
+        eeo_category = data.get("eeoCategory", {}).get("id")
+        compensation = data.get("compensation", {})
+        job["tags"] = [
+            dict(name="status", value=status),
+            dict(name="posting_status", value=posting_status),
+            dict(name="job_uuid", value=job_uuid),
+            dict(name="experience_level", value=experience_level),
+            dict(name="type_of_employment", value=employmentType),
+            dict(name="compensation", value=compensation),
+            dict(name="industry", value=industry),
+            dict(name="creator", value=creator),
+            dict(name="function", value=function),
+            dict(name="department", value=department),
+            dict(name="manual", value=manual),
+            dict(name="remote", value=remote),
+            dict(name="eeo_category", value=eeo_category),
```

Comment:
The names of tags, sections and metadatas must start with the name of the connector + "_" + the name of the object. And each name of the sub-objects is separated by "-". 

Thus 
* `posting_status` becomes `smartrecruiters_postingStatus` ("postingStatus" is the name given by Smart Recruiters actually. We don't change the nomenclature to avoid making it too complicated when there will be many connectors)
* `department` becomes `smartrecruiters_department-id`.
* `manual` becomes `smartrecruiters_location-manual`
...

Reply:
:warning: **Don't forget to change the section names**. For the section titles, I either wrote something or put the section name back in there.